### PR TITLE
feat: Shows link location and respective icon in /bookings

### LINKED ISSUE #11730: [CAL-2590] Show link location in /bookings
show link to join a call in the /booking overview

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2590](https://linear.app/calcom/issue/CAL-2590/show-link-location-in-bookings)</sub>

### PR DESCRIPTION
**Pull request by kart1ka**
## What does this PR do?

Shows link location and respective icon in /bookings.
Fixes #11730 
/claim #11730 

Screenshot from 2023-12-13 07-25-10

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

Once a booking is created, go to the /bookings page and observe the link shown in booking card.

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by Udit-takkar** [comment]
@kart1ka the bounty tag was removed from the issue. not sure why i see algora bot comment here

File: apps/web/components/booking/BookingListItem.tsx [hunk 1]
```diff
@@ -359,6 +376,33 @@ function BookingListItem(booking: BookingItemProps) {
...
+                      target="_blank"
+                      title={locationToDisplay}
+                      rel="noreferrer"
+                      className="text-sm leading-6 text-blue-600 hover:underline">
+                      <div className="flex items-center gap-2">
+                        <img
+                          src={provider?.iconUrl || ""}
+                          className="h-4 w-4 rounded-sm"
+                          alt={`${provider?.label} logo`}
+                        />
```
Comment (Udit-takkar) [inline apps/web/components/booking/BookingListItem.tsx:395]:
```suggestion
                     {provider?.iconUrl && (
                          <img
                            src={provider.iconUrl}
                            className="h-4 w-4 rounded-sm"
                            alt={`${provider?.label} logo`}
                          />
                        )}
```

File: apps/web/components/booking/BookingListItem.tsx [hunk 2]
```diff
@@ -93,6 +98,18 @@ function BookingListItem(booking: BookingItemProps) {
 
   const paymentAppData = getPaymentAppData(booking.eventType);
 
+  const location = booking.location as ReturnType<typeof getEventLocationValue>;
+  const locationVideoCallUrl: string | undefined = bookingMetadataSchema.parse(
+    booking?.metadata || {}
+  )?.videoCallUrl;
```
Comment (Udit-takkar) [inline apps/web/components/booking/BookingListItem.tsx:104]: NIT
```suggestion
  const locationVideoCallUrl = bookingMetadataSchema.parse(
    booking?.metadata || {}
  )?.videoCallUrl;
```

**Review by Udit-takkar — DISMISSED** [review]
We only want to display the url location and not address (organizer address or attendee address)

Screenshot 2023-12-14 at 12 57 40 AM

**Comment by kart1ka** [comment]
@Udit-takkar I have made the changes as per the request. Do let me know if any more changes need to be made.

**Review by Udit-takkar — APPROVED** [review]

**Comment by kart1ka** [comment]
Hi @Udit-takkar .  Is there any more changes that I need to make to get this PR merged?

**Pull request merged**